src/main.js

```diff
@@ -4,6 +4,6 @@ import router from './route';
 import './index.css';
 import axios from 'axios';
 
-axios.defaults.baseURL = 'http://127.0.0.1:8000/api';
+axios.defaults.baseURL = import.meta.env.VITE_API_BASEUR;
```

Comment:
Shouldn't it be `VITE_API_BASEURL` ?
If you add this, you need to add a `.env.example` file to the root of your project.
It will allow any developer (or you, later, when in ten years you'll git clone your project and will have to install it somewhere :-D) to have a functionnal .env file.

Take a look at this one https://github.com/heypongo/PongoDashboardVue/blob/dev/.env.example